src/backend/fold_v2.js

```diff
@@ -0,0 +1,1066 @@
+/* eslint-disable no-empty-label */
+'use strict';
+
+const moment = require('moment');
+const distHelper = require('./dist');
+const urljoin = require('url-join');
+const async = require('async');
+const timeToPixel = 50; // 15 mins = 50 pixels
+const columnWidth = 160;
+const calendarWidth = 1060;
+
+function byProperty(key) {
+  return (a, b) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Block must be padded by blank lines. (padded-blocks)](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)